### PR TITLE
plenty-426 fixing price filter  default values

### PR DESCRIPTION
**Pull request by msilinskas**

### CODE REVIEW

File: resources/js/src/app/components/itemList/filter/ItemColorTiles.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import url from '../../../mixins/url'
 import { SVGInjector } from '@tanem/svg-injector'
-import DOMPurify from 'dompurify'
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemColorTiles.js:3]: :+1:

File: resources/js/src/app/components/itemList/filter/ItemFilterPrice.js [hunk 2]
```diff
@@ -47,12 +51,18 @@ Vue.component("item-filter-price", {
...
+                    min: this.priceMin ? this.priceMin : 0,
+                    max: this.priceMax ? this.priceMax : this.getMaxPrice()
                 };
 
                 this.updateSelectedFilters(this.facet.id, facetValue);
             }
+        },
+
+        getMaxPrice() {
+            let maxPrice = this.facet.values[this.facet.values.length -1].name.split(' - ')[1];
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemFilterPrice.js:63]: Prefer `const` over `let`.

File: resources/js/src/app/components/itemList/filter/ItemFilterPrice.js [hunk 3]
```diff
@@ -47,12 +51,18 @@ Vue.component("item-filter-price", {
...
+
+            return maxPrice ? maxPrice : Number.MAX_SAFE_INTEGER;
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemFilterPrice.js:65]: Would a `maxPrice` of `0` therefore evaluate to `Number.MAX_SAFE_INTEGER`?

File: resources/js/src/app/components/itemList/filter/ItemFilterPrice.js [hunk 4]
```diff
@@ -47,12 +51,18 @@ Vue.component("item-filter-price", {
         triggerFilter() {
             if (!this.isDisabled) {
                 let facetValue = {
-                    min: this.priceMin,
-                    max: this.priceMax ? this.priceMax : Number.MAX_SAFE_INTEGER
+                    min: this.priceMin ? this.priceMin : 0,
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemFilterPrice.js:54]: Maybe it makes sense to define constants for `MIN_PRICE` and `MAX_PRICE`.

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/js/src/app/components/itemList/filter/ItemFilterPrice.js (hunk 3 above)
Comment (msilinskas) [reply resources/js/src/app/components/itemList/filter/ItemFilterPrice.js:65]: Yes, if max price is equal to 0, then Number.MAX_SAFE_INTEGER will be used.

File: package-lock.json [hunk 5]
```diff
@@ -1,9274 +1,15 @@
 {
   "name": "findologic",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```
Comment (TheKeymaster) [inline package-lock.json]: Oof you made a mistake. Please revert this file to how it was before and make sure that your npm and node versions are up to date. Here are mine (they are not on the latest versions, but will produce a version 2 lock file):

image

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**